lightkurve/targetpixelfile.py

```diff
@@ -190,6 +196,122 @@ def from_archive(target, cadence='long', quarter=None, month=None,
     def __repr__(self):
         return('KeplerTargetPixelFile Object (ID: {})'.format(self.keplerid))
 
+
+    def find_stars(self, catalog=None, radius=0.5):
+        """
+        Load tpf file to find field stars within the tpf.
+
+        Returns a crossmatched table of stars that are within the tpf file
+          given a tpf mission (either KIC or EPIC).
+
+        Parameters
+        -----------
+        catalog: string
+            Indicate catalog assigned for mission. If Kepler, catalog will be KIC
+            if K2 catalog is EPIC.
+        radius: float
+            Radius of cone search centered on the target in arcminutes.
+            Default radius is 0.5 arcmin.
+
+        Returns
+        -------
+        ID : astropy.table.column (KIC & EPIC)
+            Catalog ID from catalog.
+        RAJ200: astropy.table.column (KIC & EPIC)
+            Right ascension [degrees]
+        DEJ2000: astropy.table.column (KIC & EPIC)
+            Declination [deg]
+        pmRA: astropy.table.column (KIC & EPIC)
+            Proper motion for right ascension [mas/year]
+        e_pmRA: astropy.table.column (KIC & EPIC)
+            Error proper motion for Right ascension [mas/year]
+        pmDEC: astropy.table.column (KIC & EPIC)
+            Proper motion for declination [mas/year]
+        e_pmDEC: astropy.table.column (EPIC)
+            Error proper motion for declination [mas/year]
+        Kpmag: astropy.table.column (KIC & EPIC)
+            Magnitude in Kepler band [mag]
+        """
+
+        if catalog is None:
+            if self.mission == 'Kepler':
+                catalog = 'KIC'
+            elif self.mission == 'K2':
+                catalog = 'EPIC'
+            else:
+                log.error('Please provide a catalog.')
+
+        if catalog is "Gaia":
+            log.warn('Gaia RAs and Decs are at EPOC 2015.5. These RA/Decs have not been corrected.')
+
+        # Vizier id's
+        ID = {"KIC":
+                    {'vizier':"V/133/kic",
+                     'parameters':["KIC", "RAJ2000", "DEJ2000", "pmRA", "pmDE", "kepmag"]},
+               "EPIC":
+               {'vizier':"IV/34/epic",
+                'parameters':["ID", "RAJ2000", "DEJ2000", "pmRA", "pmDEC", "e_pmRA", "e_pmDEC", "Kpmag"]},
+               "Gaia":
+               {'vizier':"I/345/gaia2",
+               'parameters':["DR2Name", "RA_ICRS", "DE_ICRS", "pmRA", "pmDE", "e_pmRA", "e_pmDE", "Gmag"]}}
+
+        # identifies catalog
+        viz_id = (ID[catalog])['vizier']
+        pars = (ID[catalog])['parameters']
+
+        #Skycoord the centre of target
+        cent = SkyCoord(ra=self.ra, dec=self.dec, frame='icrs', unit=(u.deg, u.deg))
+
+        # Choose columns from Vizier
+        v = Vizier(catalog=[viz_id], columns=pars)
```

Comment:
This can be 

`data = Vizier(catalog=[viz_id], columns=pars)[viz_id]`

to reduce the lines below.